### ISSUE TITLE
Bug: Incorrect screen reader time readout for submission history 

### ISSUE THREAD

**Issue by victoriaatraft** [raft-tech/TANF-app#3481]
Thank you for taking the time to let us know about the issue you found. The basic rule for bug reporting is that 
something isn't working the way one would expect it to work. Please provide us with the information requested 
below and we will look at it as soon as we are able.

### Description

Apple screen readers (VoiceOver) read out **submission time** in the **Submission History Table** incorrectly. 

Currently, the time is read as **"9 Hours and 25 Minutes 35 Seconds AM"** instead of a natural time format like **"Nine Twenty Five AM"**. 

Further research is needed to identify a solution that ensures accessible, screen-reader-friendly date and time formatting for the submission history report.

#### Duplicate the Issue:
- Use the screen reader (`Command-F5` on Mac)
- Navigate over the `Submitted` column cell in the **Submission History Table**
- Screen reader reads out: **"9 Hours and 25 Minutes 35 Seconds AM"**
- Expected: **"Nine Twenty Five AM"**

#### Supporting Screenshot:
<img width="898" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNVg5Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--60c99da5bc345ceedfe227577fd9dc3a5d8fb833/Screenshot%202025-02-24%20at%207.02.41%20AM.png" alt="Screenshot 2025-02-24 at 7.02.41 AM.png" />

#### Supporting Resource:
- [GitHub: Readable, accessible dates for screen readers](https://github.com/MikesBarto/accessible-date)

### Action Taken

- Triggered VoiceOver (`Command-F5`) on a Mac
- Navigated through the **Submission History Table**
- Focused on the `Submitted` column
- Noted the screen reader output

### What I expected to see

The screen reader should read the submitted time in a natural, speech-friendly format:
> "Nine Twenty Five AM"

### What I did see

The screen reader reads the time as:
> "9 Hours and 25 Minutes 35 Seconds AM"

### Other Helpful Information

+ URL of the page I was on: Submission History Table page
+ Browser and version: _TBD_
+ Operating System: MacOS X
+ Is the issue repeatable?: yes
+ Has the issue occurred more than once?: yes

### Usability Testing Criteria
_PM/UX/OFA Only: Create a list of expected user behaviors that should be confirmed when UX and/or PM is testing this ticket_
- [ ] Screen reader reads **submission time** in a natural format (e.g., "Nine Twenty Five AM")
- [ ] Time values do not get read as "Hours" and "Minutes"
- [ ] Solution is accessible and consistent across browsers with screen reader use

**Comment by lhuxraft** [comment]
@victoriaatraft Can we build out this ticket a bit more? I don't think I have enough information to understand scope

**Comment by lhuxraft** [comment]
This should apply to all instances of submission history (FRA and TANF/SSP)
